app/controllers/foo_controller.rb

```diff
@@ -0,0 +1,13 @@
+class FooController < ActionController::Base  
+  def new_user
+    user = User.first(:conditions => ("username = '#{params[:username]}'"))
+    user = User.first(:conditions => ("username = '#{params[:username]}'"))
+    user = User.first(:conditions => ("username = '#{params[:username]}'"))
+    if user.blank?
+      render text: user.username
+      return
+    end
+
+    render text: "test"
```

Comment:
__[staging][rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->